Shared/ContextMenuController.h

```diff
@@ -110,8 +111,8 @@ private slots:
   Esri::ArcGISRuntime::TaskWatcher m_identifyFeaturesTask;
   Esri::ArcGISRuntime::TaskWatcher m_identifyGraphicsTask;
   Esri::ArcGISRuntime::TaskWatcher m_screenToLocationTask;
-  QHash<QString, QList<Esri::ArcGISRuntime::GeoElement*>> m_contextFeatures;
-  QHash<QString, QList<Esri::ArcGISRuntime::GeoElement*>> m_contextGraphics;
+  QMultiHash<QString, QList<Esri::ArcGISRuntime::GeoElement*>> m_contextFeatures;
```

Comment:
This is needed because QHash.unite was deprecated. https://doc.qt.io/qt-5/qhash-obsolete.html#unite

Reply:
The rest of the code needs to assume there can be duplicates now. Are you sure this is correct?

Reply:
I found that QHash::insert makes more sense than refactoring to use QMultiHash everywhere. Both compiled and appeared to work, but the MultiHash seems to open it up for more to go wrong